bin/nbp/nbplkup.c

```diff
@@ -132,29 +132,20 @@ int main(int ac, char **av)
             exit(1);
         }
 
-        name = (char *)malloc(strlen(Obj) + 1);
-        if (name == NULL) {
-            perror("malloc");
-            exit(1);
+        Obj = strndup(Obj, 32);
+        if (Obj == NULL) {
+            perror("strndup(Obj)");
```

Comment:
You don't think we need to exit the program anymore when this check fails? I'm genuinely curious if you perhaps saw that error handling is robust enough elsewhere. :)

Reply:
No, that was a copy and paste fail on my part, not any indication of confidence... this is why code review is a good thing.  One moment please, and I will fix.